src/metrics/basic.py

```diff
@@ -85,9 +86,17 @@ def process(l):
         return '({})'.format(' OR '.join(arr))
     repoWhereClauseArray = []
     if config.get('repoIds'): repoWhereClauseArray.append('repo_id IN {}'.format(config.get('repoIds')))
-    if config.get('repoNames'): repoWhereClauseArray.append('repo_name IN {}'.format(config.get('repoNames')))
+    if config.get('repoNames'):
```

Comment:
As all names for repo, org and user will be used in group function, so we need to get id first and pass into where clause rather than use names in where clause.